src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TabType } from '../../types';
+
+export const Tab = ({ index, onTableSelected, title, isActive }) => (
+  <button
+    type="button"
+    onClick={() => onTableSelected(index)}
+    className={isActive ? 'button' : 'button is-light'}
```

Comment:
good, but also you can use library `classnames`

Reply:
Good advice